lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -287,7 +287,9 @@ def forum_form_discussion(request, course_key):
             'is_course_cohorted': is_course_cohorted(course_key),  # still needed to render _thread_list_template
             'sort_preference': user.default_sort_key,
             'category_map': course_settings["category_map"],
-            'course_settings': json.dumps(course_settings)
+            'course_settings': json.dumps(course_settings),
+            'disable_courseware_js': True,
```

Comment:
@cahrens That's an interesting point. My thinking was that we use ``disable_courseware_js`` in Teams which also renders the discussion component, so I couldn't think why it would be a problem here. My preference would be to try it like this because it greatly reduces the weight of the page, and address any issues that come up. Does that sound okay?